src/lib.rs

```diff
@@ -989,12 +989,16 @@ mod tests {
 
     #[test]
     fn load_symbolfilecfg_processkernel() {
+        let kallsyms = Path::new(&env!("CARGO_MANIFEST_DIR"))
+            .join("data")
+            .join("kallsyms");
+
         // Check if SymbolSrcCfg::Process & SymbolSrcCfg::Kernel expands to
         // ELFResolvers and a KernelResolver.
         let srcs = vec![
             SymbolSrcCfg::Process { pid: None },
             SymbolSrcCfg::Kernel {
-                kallsyms: None,
+                kallsyms: Some(kallsyms),
                 kernel_image: None,
```

Comment:
that checked in kallsyms.xz matches `/data/vmlinux-5.17.12-100.fc34.x86_64`, right? Should this pass kernel image path as well?

Reply:
> that checked in kallsyms.xz matches /data/vmlinux-5.17.12-100.fc34.x86_64, right?

It actually does not. There is no relationship between the two at this point.

> Should this pass kernel image path as well?

I am just adjusting the test without changing its meaning. `kallsyms: None` mapped to using `/proc/kallsyms`, which may not exist. By redirecting we make sure that it does in fact exist. We have a similar test below that does set `kernel_image` (though to an invalid value).